src/triedb.rs

```diff
@@ -147,6 +145,67 @@ impl<'db> Trie for TrieDB<'db> {
     }
 }
 
+impl<'db> CryptoStructure for TrieDB<'db> {
+    /// A proof creation logic for TrieDB.
+    /// A proof is basically a list of serialized trie nodes, Vec<Bytes>.
+    /// It starts from the one closest to the root, and to the leaf. (It may not reach the leaf in absence case)
+    /// Each node can be decoded with RLP. (Note that RLP doesn't guarantee format detail, so you must check our serialization code)
+    /// In case of precense, the list will contain path from the root to the leaf with the key.
+    /// In case of absence, the list will contain path to the last node that matches the key.
```

Comment:
In case of absence, the list will contain a path to the last node that matches the key.